tools/dockerbuild/docker/Dockerfile-jvm

```diff
@@ -34,5 +34,4 @@ COPY --chown=185 build/quarkus-app/quarkus/ /deployments/quarkus/
 EXPOSE 19120
 USER 185
 ENV AB_JOLOKIA_OFF=""
-ENV JAVA_OPTS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
```

Comment:
Do we not need the `-Djava.util.logging.manager` option or is it set somewhere else?

Reply:
I don't think so? I don't see any difference in logging with or without it.

Reply:
Apparently setting this property is only required for tests:

https://quarkus.io/guides/logging#configure-logging-for-quarkustest